src/components/ValueChange/ValueChange.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Label, Icon } from '@santiment-network/ui'
+import styles from './ValueChange.module.scss'
+
+const Change = {
+  true: ['jungle-green', 'triangle-up'],
+  false: ['persimmon', 'triangle-down']
+}
+
+const notChanged = ['texas-rose', 'lock-small']
+
+const ValueChange = ({ oldValue, newValue, render }) => {
+  const amp = newValue - oldValue
+  const [accent, triangle] = amp !== 0 ? Change[amp > 0] : notChanged
```

Comment:
Explicit maybe better from fast readability than 2 vars and game with types.
I mean {
positive:
negative:
notChanged:
}